node/public/JS/bmi-calc.js

```diff
@@ -0,0 +1,8 @@
+function BMI() {
+    var h = document.getElementById('h').value;
+    var w = document.getElementById('w').value;
+    var bmi = w / (h / 100 * h / 100);
```

Comment:
consider adding some validation here and/or at UI

Reply:
made changes